modules/client-handler/src/integrationTest/java/org/apache/ignite/client/handler/ItClientHandlerTest.java

```diff
@@ -159,7 +159,7 @@ void testHandshakeValidReturnsSuccess() throws Exception {
             unpacker.skipValue(extensionsLen);
 
             assertArrayEquals(MAGIC, magic);
-            assertEquals(98, len);
+            assertEquals(99, len);
```

Comment:
```suggestion
            assertEquals(99, len);
```
Can we replace expected value with the expression depending on `extensionLen` ?

Reply:
done